### PR TITLE
Add institutional overlays management

### PR DESCRIPTION
**Pull request by danielkryska**
- Add manage overlays button into `/settings/institution?institution=...` (institution profile)

### CODE REVIEW

File: s4e-web/src/app/components/overlay-list/overlay-list.component.html [hunk 1]
```diff
@@ -0,0 +1,241 @@
+<ng-container *ngIf="!(isLoading$ | async); else loadingRef">
+  <div *ngIf="!(overlays$ | async) || (overlays$ | async).length === 0" class="infobox infobox--emptylayers">
+    <p>
+      Nie masz jeszcze żadnej swojej nakładki. <a href="#">Sprawdź czym są nadkładki.</a>
```
Comment (michal-szostak) [inline s4e-web/src/app/components/overlay-list/overlay-list.component.html:4]: Where does this link lead?

**Review by michal-szostak — APPROVED** [review]

File: s4e-web/src/app/components/overlay-list/overlay-list.component.html (hunk 1 above)
Comment (danielkryska) [reply s4e-web/src/app/components/overlay-list/overlay-list.component.html:4]: I don't know... I belive Andrzej would know

**Comment by Marcelinna** [comment]
It is not possible to add two layers with the same name because I get error 404

**Comment by danielkryska** [comment]
> It is not possible to add two layers with the same name because I get error 404

It's backend stuff

**Comment by Marcelinna** [comment]
Although admin@mail.pl in Global WMS will turn off the visibility of layers, they are visible in all accounts

**Comment by Marcelinna** [comment]
After adding layer, it isnt visible 
image

**Comment by Marcelinna** [comment]
> Although [admin@mail.pl](mailto:admin@mail.pl) in Global WMS will turn off the visibility of layers, they are visible in all accounts

This same situation for user with administrator privileges when they turn off layer for layer in /settings/institution-wms-overlays?

**Review by jswk — APPROVED** [review]
Global WMS shouldn't be exposed in the UI. They are to be API managed, as you need to reconfigure several components when modifying them.

**Review by kmarszalek — APPROVED** [review]

**Pull request merged**